.github/workflows/sync-local-fallback-files.yml

```diff
@@ -0,0 +1,72 @@
+name: Sync runtime local fallback files
+on:
+  schedule:
+    # https://crontab.guru/every-night-at-midnight
+    - cron: "0 0 * * *"
+
+jobs:
+  sync-local-fallback-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          extensions: dom, iconv, json, libxml, zip
+          tools: composer
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Configure Composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        run: composer install
+
+      - name: Create branch with committed changes
+        run: |
+          git config --global user.email "alain.schlesser@gmail.com"
+          git config --global user.name "Alain Schlesser"
+          git fetch
+          git branch -f sync-local-fallback-files origin/main
+          git checkout sync-local-fallback-files
+          git pull --no-rebase || true
```

Comment:
Is the `git pull` needed here? You just above created a new branch from `origin/master` after you `git fetch`, so then there should never be any new changes, right?

```suggestion
```

Reply:
The reason I have this in here is because:
- There might already be a branch with this name, which should be updated, as I don't want to script to create multiple PRs if we don't get to merge them right away: and
- On an existing PR, we will sometimes add manual commits to fix the tests that break with an automated update to make them pass again, so the script should only ever _add_ to an existing PR, and not _replace_ it and cause manual work to be lost.

Reply:
Thanks. Makes sense.